### ISSUE TITLE
"curl https://piku.github.io/get | sh" gets the piku-bootstrap file from piku-bootstrap repo which does not contain install-cli command

### ISSUE THREAD

**Issue by varianone** [piku/piku#193]
First of all thank you for creating this project!

It looks like there are two different versions of the piku-bootstrap file. one under piku repo and one under piku-bootstrap repo.

The readme in piku repo provides command "curl https://piku.github.io/get" in order to get the piku-bootstrap file which gets it from the piku-bootstrap repo.
`$ curl https://piku.github.io/get
#!/bin/sh

set -e

echo "Downloading piku-bootstrap here."
curl -s https://raw.githubusercontent.com/piku/piku-bootstrap/master/piku-bootstrap > piku-bootstrap
chmod 755 piku-bootstrap
echo "Now you can install Piku on `hostname` like this:"
echo "./piku-bootstrap install"`

after that the documentation mentions to run:
`./piku-bootstrap install-cli ~/bin`

However the file version under piku-bootstrap repo is missing this option, while version under piku repo does have it, although based on commits it looks to be newer.

Which is the correct one?

**Comment by chr15m** [comment]
@varianone many users had trouble with the old bootstrap script running on their local machines, so I made a new version which runs on the remote machine instead (so you ssh in as root first and then run the commands above). If the script is running on a remote machine then the `install-cli` command can't work, because that cli is supposed to run on your local. I hope to finish off the transition to the new bootstrap script soon and then I'll submit a PR to update the documentation and remove the old bootstrap scripts, for @rcarmo to review.

Let me know if you have any thoughts on this.

**Comment by varianone** [comment]
@chr15m this sounds like a good clean approach, the installation process is done via running the commands on the remote machine directly.  One thought here is that not always root login is enabled by default like on the Raspberry PI. So supporting install with sudo out of the box may be a good idea.
I attempted the install using sudo and the install failed with ansible error that it cant find the authorized_keys file under /root/.ssh. I created the file manually and the installed completed successfully.

for cli tools, I guess there should be a separate command using "curl url | sh" for installing on local.
